model/Core/Classes/NamespaceMap.md

```diff
@@ -0,0 +1,66 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NamespaceMap
+
+## Summary
+
+A mapping between prefixes and namespace URIs.
+
+## Description
+
+A namespace map allows the creator of a collection of Elements to suggest 
+a set of shorter identifiers ("prefixes") for particular namespace portions 
+of ElementIDs to be used in SPDX content serialization in order to provide a more
+human-readable and smaller serialized representation of the Elements.
+
+The content of a namespace map are suggestions for consistency in serialization but are not normatively enforced.
+Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content.
+A serialization MAY choose to use the namespace map content.
+A serialization MAY choose to use prefixes and namespaces other than the namespace map content.
+A serialization MAY choose to use no prefixes at all and rather use the more verbose full ElementID IRIs.
+
+If utilized the namespace map set of prefixes and namespaces MUST be implemented in a given serialization 
+form (e.g., json-ld or xml) as specified in the binding rules specification for that serialization and 
+utilizing the appropriate inherent or custom specified mechanism for that serialization.
+The namespace map itself is also conveyed as native SPDX content to support clarity, transparency and 
+consistency independent of any particular serialization form.
+
+A given serialization payload (whether file or streaming) MUST NOT contain multiple namespace maps with conflicting mappings.
```

Comment:
I take it the above sentence refers to the "native" namespace mapping.  If so, this may not be a required statement since the serialization format standards already have this requirement.

Reply:
Serialization formats that natively support prefixes usually have this requirement though some do not enforce it and rather simply utilize the last defined prefix and ignore any earlier conflicting prefixes.
Serialization formats that do not natively support prefixes and where we will have to define custom prefix representations will have no such rules.

In either case, I would propose making this explicit statement is highly useful in the SPDX spec.